src/fastoad_cs25/models/geometry/geom_components/wing/components/compute_center_chord.py

```diff
@@ -0,0 +1,93 @@
+"""
+Estimation of center wing chord (l1')
+"""
+#  This file is part of FAST-OAD : A framework for rapid Overall Aircraft Design
+#  Copyright (C) 2025 ONERA & ISAE-SUPAERO
+#  FAST is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import numpy as np
+import openmdao.api as om
+from scipy.interpolate import interp1d
```

Comment:
as explained in scipy's documentation, interp1D is a legacy function: https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.interp1d.html
While it doesn't mean it will be removed just yet, it won't receive updates.